sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md

```diff
@@ -2987,6 +2988,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -ContentSecurityPolicyConfigurationSynced
```

Comment:
- I would call it something in the line of `ResyncContentSecurityPolicyConfigurationEntries" and make it == TRUE